### ISSUE TITLE
Compatibility with psr/log 3.0

### ISSUE THREAD

**Issue by hubertnnn** [exonet/powerdns-php#109]
**Describe the bug**
This package depends on psr/log ^1.0 but current version is 3.0.
Installing will cause this error:
```
  Problem 1
    - Root composer.json requires exonet/powerdns-php ^4.1 -> satisfiable by exonet/powerdns-php[v4.1.0].
    - exonet/powerdns-php v4.1.0 requires psr/log ^1.0 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

**To Reproduce**
```
# Install latest psr/log or a package that depends on it.
composer require psr/log 3.0.0
composer require exonet/powerdns-php
```

**Expected behavior**
Package installs

**Comment by robbinjanssen** [comment]
Hi @hubertnnn,

Feel free to submit a PR, when we have time we'll see if we can easily update this to support `v1 || v2 || v3` :)

**Comment by trizz** [comment]
Hi @hubertnnn,

I've just released [version 4.1.1](https://github.com/exonet/powerdns-php/releases/tag/v4.1.1) that will fix this.
